chaincode-labs/chaincode-podcast/block-building.md

```diff
@@ -207,94 +213,96 @@ You could have an invalid script or invalid signature, or you could have a trans
 So if you naively just check that a transaction only spends one set of inputs and no other transaction does, you might miss that you spent the same funds twice.
 So I don't know the whole set of rules by heart, but there's a bunch of them.
 
-Speaker 2: 00:08:49
+Clara Shikhelman: 00:08:49
 
 So, in this space of block building, I think we'll mostly focus on the limit on the block size and only spending available UTXOs or existing UTXOs.
 
-Speaker 0: 00:09:04
+Patrick Murck: 00:09:04
 
 Maybe one more comment.
 There's a special transaction that has to be in every block, which is the coinbase transaction or the generating transaction.
 It's the one that goes first in the block and that can collect a transaction fee and create new bitcoins.
-And after that, a miner may include zero to a full block of transactions, and they can pick any transactions they want, as long as they follow these rules that They're not spending any funds that don't exist yet and so forth.
+And after that, a miner may include zero to a full block of transactions, and they can pick any transactions they want, as long as they follow these rules that they're not spending any funds that don't exist yet and so forth.
 It's completely up to the miner to pick what they want to include, but generally is the goal of the miner to collect as much money as possible.
 So we expect miners to build the most profitable block.
 
-Speaker 2: 00:09:46
+Clara Shikhelman: 00:09:46
 
 So how would they do that?
 
-Speaker 0: 00:09:48
+Patrick Murck: 00:09:48
 
 Well, currently we assume that most miners are using Bitcoin Core and Bitcoin Core has a function called get block template.
```

Comment:
You missed this "get block template".
`getblocktimestamp` is a technical term and should be enclosed in backticks to align with our transcription style.
Please review the transcript for these AI errors and correct them